test/y2partitioner/widgets/menus/view_test.rb

```diff
@@ -26,21 +26,29 @@
 require "y2partitioner/widgets/menus/view"
 
 describe Y2Partitioner::Widgets::Menus::View do
-  before do
-    allow(Y2Partitioner::Dialogs::DeviceGraph).to receive(:supported?).and_return(graphical)
-  end
-
   subject(:menu) { described_class.new }
 
   let(:graphical) { true }
 
+  let(:issues_manager) { instance_double(Y2Storage::IssuesManager, probing_issues: issues) }
+  let(:devicegraph) { instance_double(Y2Storage::Devicegraph, issues_manager: issues_manager) }
+  let(:device_graphs) { instance_double(Y2Partitioner::DeviceGraphs, system: devicegraph) }
+
+  let(:issues) { Y2Issues::List.new }
+
+  before do
+    allow(Y2Partitioner::Dialogs::DeviceGraph).to receive(:supported?).and_return(graphical)
+    allow(Y2Partitioner::DeviceGraphs).to receive(:instance).and_return(device_graphs)
+  end
+
   include_examples "Y2Partitioner::Widgets::Menus"
 
   describe "#items" do
     it "includes entries for installation summary, settings and csets" do
```

Comment:
the description is now incomplete :(

We could change it to 

```suggestion
    it "includes entries for installation summary, settings, Bcache csets, and system issues" do
```

but maybe is  time to use a generic one.